scripts/install-helpers/baremetal-coco/intel-dcap/pccs.yaml.in

```
@@ -0,0 +1,123 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: pccs-config
+  namespace: intel-dcap
+data:
+  default.json: |
+    {
+      "HTTPS_PORT" : 8042,
+      "hosts" : "0.0.0.0",
+      "uri": "https://api.trustedservices.intel.com/sgx/certification/v4/",
+      "ApiKey" : "${PCCS_API_KEY}",
+      "proxy" : "${HTTP_PROXY}",
+      "RefreshSchedule": "0 0 1 * * *",
+      "UserTokenHash" : "${PCCS_USER_TOKEN_HASH}",
+      "AdminTokenHash" : "${PCCS_ADMIN_TOKEN_HASH}",
+      "CachingFillMode" : "LAZY",
+      "OPENSSL_FIPS_MODE" : false,
+      "LogLevel" : "debug",
```

The PCCS default is:
```suggestion
      "LogLevel" : "info",
```